index.rst

```diff
@@ -4,20 +4,715 @@
+
+Requirements satisfied: [...]
+
+.. _daily-cpp-payload:
+
+Daily calibration products update payload
+-----------------------------------------
+
+Executes under control of the OCS-controlled batch service so that its execution can be synchronized with the observing schedule.
+Uses raw calibration images and information from the Transformed EFD to generate a subset of Master Calibration Images and Calibration Database entries in the Data Backbone.
+
+Requirements satisfied: [...]
+
+.. _intermediate-cpp-payload:
+
+Intermediate-period calibration products production payloads
+------------------------------------------------------------
+
+Execute under control of the CPP Execution service at nominally monthly intervals but perhaps as frequently as weekly or as infrequently as quarterly, depending on the stability of Observatory systems and their calibrations.
+Uses raw calibration images and information from the Transformed EFD to generate a subset of Master Calibration Images and Calibration Database entries in the Data Backbone.
+
+.. _template-generation-payload:
+
+Template generation payload
+---------------------------
+
+Executes under control of the CPP Execution service if necessary to generate templates for Alert Production in between annual Data Release Productions.
+Uses raw science images to generate the templates, placing them in the Data Backbone.
+
+.. _annual-cpp-payload:
+
+Annual calibration products production payload
+----------------------------------------------
+
+Executes under control of the CPP Execution service at annual intervals prior to the start of the Data Release Production.
+Uses raw calibration images, information from the Transformed EFD, information from the Auxiliary Telescope Spectrograph, and external catalogs to generate Master Calibration Images and Calibration Database entries in the Data Backbone.
+
+.. _drp-payload:
+
+Data Release Production payload
+-------------------------------
+
+Executes under control of the DRP Execution service at annual intervals, first running a "mini-DRP" over a small portion of the sky, followed by the full DRP over the entire sky.
+Produces science data products in the Data Backbone.
+
+
+.. _suit:
+
+SUIT
+====
+
+The Science User Interface and Tools provide visualization, plotting, catalog rendering, browsing, and searching elements that can be assembled into predetermined "portals" but can also be used flexibly within dynamic "notebook" environments.
+
+
+.. _middleware:
+
+Middleware
+==========
+
+.. _middleware-data-butler:
+
+Data Butler access client
+-------------------------
+
+The Data Butler provides an access abstraction for all science payloads that enables their underlying data sources and destinations to be configured at runtime with a variety of back-ends ranging from local disk to network locations and a variety of serializations ranging from YAML and FITS files (potentially with the addition of HDF5 or ASDF) to database tables.
```

Comment:
Is Science Platform code is considered part of "all science payloads" here?

I'm guessing it can utilize *either* the Butler or "Data Access Web Services", but it's not allowed to talk to the Data Backbone directly?